npm/vue/examples/code-coverage/cypress.config.js

```diff
@@ -1,15 +1,13 @@
+const { devServer } = require('@cypress/webpack-dev-server')
+
 module.exports = {
   'fixturesFolder': false,
   'video': false,
   'component': {
     devServer (cypressDevServerConfig) {
-      const { startDevServer } = require('@cypress/webpack-dev-server')
       const webpackConfig = require('./webpack.config')
 
-      return startDevServer({
-        options: cypressDevServerConfig,
-        webpackConfig,
-      })
+      return devServer(cypressDevServerConfig, webpackConfig)
```

Comment:
Should this be

```suggestion
      return devServer(cypressDevServerConfig, {webpackConfig })
```

Also, see my first comment.